src/Components/Medicine/PrescriptionDetailCard.tsx

```diff
@@ -28,7 +36,7 @@ export default function PrescriptionDetailCard({
         prescription.discontinued && "bg-gray-200 opacity-80",
       )}
     >
-      <div className="flex flex-1 flex-col">
+      <div className="flex flex-1 flex-col" onClick={handleShow}>
```

Comment:
The user should know that the element is clickable.

```suggestion
      <div className="flex flex-1 flex-col cursor-pointer" onClick={handleShow}>
```

Reply:
Also, can we add animations on hover for the card since it's a clickable element?